### PR TITLE
Random emoji support

### PR DESCRIPTION
**Pull request by lramos15**
Added a JSON file with support for random emojis. Also divided sentiment between happy, medium, and sad.

### CODE REVIEW

File: utilities/emoji.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "happy": [
+        ":smile:",
+        ":partyparrot:",
+        ":logan_cheesin:",
+        ":dan:",
```
Comment (ddbruce) [inline utilities/emoji.json]: I'm not sure this is "happy," and I'd prefer it to go away altogether

File: utilities/helperFunctions.js [hunk 2]
```diff
@@ -75,6 +77,27 @@ const updateModMessage = (status, channel_id, text, user_id, ts, moderator) => {
     });
     return;
   }
+
+  const randomEmoji = (sentiment) => {
+    let emojis = [];
+    switch(sentiment){
```
Comment (ddbruce) [inline utilities/helperFunctions.js]: Please run this through ESLint. There should be a space between the `)` and the `{` if it's running correctly.

File: utilities/helperFunctions.js [hunk 3]
```diff
@@ -75,6 +77,27 @@ const updateModMessage = (status, channel_id, text, user_id, ts, moderator) => {
...
+      case "happy":
+        emojis = emojisList.happy;
+        break;
+      case "medium":
+        emojis = emojisList.medium;
+        break;
+      case "sad":
+        emojis = emojisList.sad;
+        break;
+      default:
```
Comment (ddbruce) [inline utilities/helperFunctions.js]: Correct me if I'm wrong, but this means that if a sentiment isn't specified, it just picks a random one?

File: utilities/emoji.json (hunk 1 above)
Comment (lramos15) [reply utilities/emoji.json]: You're happy and :dan: will last as long as :logan_cheesin: but it's simple JSON and can be removed

Comment (lramos15) [reply utilities/emoji.json]: You're happy and :dan: will last as long as :logan_cheesin: but it's simple JSON and can be removed

File: utilities/helperFunctions.js (hunk 2 above)
Comment (lramos15) [reply utilities/helperFunctions.js]: Hmm I thought I had it on but I will double check

File: utilities/helperFunctions.js (hunk 3 above)
Comment (lramos15) [reply utilities/helperFunctions.js]: Yeah it picks a random emoji out of the entire JSON file

**Comment by lramos15** [comment]
It's possible to specify custom but I figured I would leave it up to deployers to create their own JSON emoji list

**Comment by ddbruce** [comment]
In that case, let's only include an emojis.json.example file

**Comment by ddbruce** [comment]
I think the `sendRejectionDm` emoji is still hard-coded, but we can change that later.

**Pull request merged**